lib/animina/accounts/resources/photo.ex

```diff
@@ -195,12 +196,19 @@ defmodule Animina.Accounts.Photo do
              {:ok, record}
            end),
            on: [:create, :destroy]
-  end
 
-  validations do
-    validate {Validations.AboutPhoto, story: :story_id, user: :user_id}, on: :destroy
+    change after_action(fn changeset, record, _ ->
+             delete_photo_and_optimized_photos(record)
+
+             {:ok, record}
+           end),
+           on: [:destroy]
   end
 
+  # validations do
```

Comment:
Why don't we have validations? How can we ensure that a `story` exists and that the `story_id` is set?